Maple2.File.Parser/TableParser.cs

```diff
@@ -102,6 +112,16 @@ public class TableParser {
         this.setItemInfoSerializer = new XmlSerializer(typeof(SetItemInfoRoot));
         this.setItemOptionSerializer = new XmlSerializer(typeof(SetItemOptionRoot));
         this.titleTagSerializer = new XmlSerializer(typeof(TitleTagRoot));
+        this.individualItemDropSerializer = new XmlSerializer(typeof(IndividualItemDropRoot));
+        this.individualItemDropChargeSerializer = new XmlSerializer(typeof(IndividualItemDropRoot));
+        this.individualItemDropEventSerializer = new XmlSerializer(typeof(IndividualItemDropRoot));
+        this.individualItemDropEventNpcSerializer = new XmlSerializer(typeof(IndividualItemDropRoot));
+        this.individualItemDropNewGachaSerializer = new XmlSerializer(typeof(IndividualItemDropRoot));
+        this.individualItemDropPetSerializer = new XmlSerializer(typeof(IndividualItemDropRoot));
+        this.individualItemDropQuestObjSerializer = new XmlSerializer(typeof(IndividualItemDropRoot));
+        this.individualItemDropQuestMobSerializer = new XmlSerializer(typeof(IndividualItemDropRoot));
+        this.individualItemDropGachaSerializer = new XmlSerializer(typeof(IndividualItemDropRoot));
+        this.individualItemDropGearBoxSerializer = new XmlSerializer(typeof(IndividualItemDropRoot));
```

Comment:
Do I need to make a serializer for each if it uses the same type?

Reply:
Nope